### PR TITLE
CodeGenerators: Return Optional from JsonObject::get() methods (PART 2)

### PR DESCRIPTION
**Pull request by AtkinsSJ**
Part 2 of https://github.com/SerenityOS/serenity/pull/16627

Prepare yourself for the white-knuckle thrill of repetitive changes. :scream:

### CODE REVIEW

**Comment by AtkinsSJ** [comment]
Rebased on master and reduced down to the CodeGenerators commit because that's chunky enough on its own.

File: Userland/Libraries/LibWebView/DOMTreeModel.h [hunk 1]
```diff
@@ -51,8 +51,8 @@ class DOMTreeModel final : public GUI::Model {
 
     ALWAYS_INLINE static JsonArray const* get_children(JsonObject const& o)
     {
-        if (auto const* maybe_children = o.get_ptr("children"sv); maybe_children)
-            return &maybe_children->as_array();
+        if (auto maybe_children = o.get_array("children"sv); maybe_children.has_value())
+            return &maybe_children.value();
```
Comment (trflynn89) [inline Userland/Libraries/LibWebView/DOMTreeModel.h:55]: Feel a little weird to change `AccessibilityTreeModel::get_children` to return a `Optional<JsonArray const&>` but not this one

**Review by trflynn89 — COMMENTED** [review]
One minor comment, LGTM though

File: Userland/Libraries/LibWebView/DOMTreeModel.h (hunk 1 above)
Comment (AtkinsSJ) [reply Userland/Libraries/LibWebView/DOMTreeModel.h]: Sure, I'll change that too.

**Review by trflynn89 — APPROVED** [review]

**Pull request merged**